src/MagnumPlugins/GltfImporter/Test/mesh-morph-target-attributes.gltf

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
To fix the CI failures, the `mesh-morph-target-attributes.gltf` file needs to be listed in among the others in the CMakeLists.

Also, I don't think anything uses the `*.glb`, remove that one.